app/views/shared/_footer_lite.html.erb

```diff
@@ -1,6 +1,9 @@
 <footer class="footer">
   <%= new_tab_link_to('https://www.gsa.gov', class: 'footer__agency-logo') do %>
-    <%= image_tag(asset_url('sp-logos/square-gsa.svg'), size: 20, alt: '', class: 'margin-right-05', aria: { hidden: true }) %>
+    <%= picture_tag(aria: { hidden: true }, class: 'margin-right-05') do %>
+      <%= tag.source(media: '(min-width: 1024px)', srcset: asset_url('sp-logos/square-gsa.svg')) %>
+      <%= image_tag(asset_url('sp-logos/square-gsa-dark.svg'), size: 20, alt: '') %>
```

Comment:
There's a little bit of trickery here, where we never expect the "fallback" to be loaded here. But the fallback is strictly required by the spec. The only reason the fallback would be triggered is for mobile, where we'd want to show the mobile image, hence the inversion between the two images here.